### ISSUE TITLE
代码问题

### ISSUE THREAD

**Issue by wiluen** [dragen1860/MAML-Pytorch#78]
for i in range(task_num):

            # 1. run the i-th task and compute loss for k=0
            logits = self.net(x_spt[i], vars=None, bn_training=True)
            loss = F.cross_entropy(logits, y_spt[i])
            grad = torch.autograd.grad(loss, self.net.parameters())
            fast_weights = list(map(lambda p: p[1] - self.update_lr * p[0], zip(grad, self.net.parameters())))

            # this is the loss and accuracy before first update
            with torch.no_grad():
                # [setsz, nway]
                logits_q = self.net(x_qry[i], self.net.parameters(), bn_training=True)
                loss_q = F.cross_entropy(logits_q, y_qry[i])
                losses_q[0] += loss_q

                pred_q = F.softmax(logits_q, dim=1).argmax(dim=1)
                correct = torch.eq(pred_q, y_qry[i]).sum().item()
                corrects[0] = corrects[0] + correct
                
  在每个任务的第一轮，      logits_q = self.net(x_qry[i], self.net.parameters(), bn_training=True)为什么用的网络参数是  `self.net.parameters()`而不是`fast_weights`呢？这两个数据的值并不一样。而后续的更新用的一直是`fast_weights`
